niworkflows/workflows/epi/refmap.py

```diff
@@ -6,80 +6,108 @@
 from nipype.interfaces import utility as niu
 
 from ...engine.workflows import LiterateWorkflow as Workflow
-from ...interfaces.fixes import (
-    FixHeaderRegistration as Registration,
-    FixHeaderApplyTransforms as ApplyTransforms,
-    FixN4BiasFieldCorrection as N4BiasFieldCorrection,
-)
-from ...interfaces.images import ValidateImage, EstimateReferenceImage
 
 
 DEFAULT_MEMORY_MIN_GB = 0.01
 
 
-def init_func_reference_wf(bold_files, omp_nthreads, multiecho=False, name='func_reference_wf'):
+def init_func_reference_wf(omp_nthreads, name="func_reference_wf"):
```

Comment:
This might be trivial, but do we want to call it `init_func_reference_wf` and not something more general (`init_epi_reference_wf`) if it's going to be for all modalities? or is it going to break a bunch of things if we change the name?

Reply:
I had the changes unstaged in my workspace when you posted this :D

Reply:
Great minds! 💡